lib/web/apiserver_test.go

```diff
@@ -606,8 +607,15 @@ func (s *WebSuite) TestPasswordChange(c *C) {
 }
 
 func (s *WebSuite) TestWebSessionsRenew(c *C) {
```

Comment:
optional: rewrite this test following https://github.com/gravitational/teleport/blob/master/rfd/0001-testing-guidelines.md if you have the time

Reply:
Done although it turned into a bit more changes than anticipated: https://github.com/gravitational/teleport/pull/5276/commits/308177f1d15865a52ccec7b6452c3171cae05228 and https://github.com/gravitational/teleport/pull/5276/commits/4ba1f5396aaf467da1b1ad39f96cf4d633aed366